### PR TITLE
Update home.ko.yml_version 1

### PR DESCRIPTION
**Pull request by misunrina**
Could you please check if this is the correct way of translating the content?

### CODE REVIEW

**Comment by mgmodell** [comment]
I will take a look at this and the previous in relation to #25 as soon as I am able.

**Comment by misunrina** [comment]
I have entered more content for translation on GitHub. I hope I have
utilized this system correctly.


Misun Modell

File: config/locales/home.ko.yml [hunk 1]
```diff
@@ -22,222 +22,216 @@
...
-    (2-inf){You've been invited to participate in the following {{count}} courses:}
-  course_invite_close: Please log in to %{site_link} accept or decline this enrollment.
+    (1){다음 수업에 참여하도록 초대되었습니다:};
+    (2-inf){다음 {{count}}개의 수업에 참여하도록 초대되었습니다:}
+  course_invite_close: %{site_link}에 로그인하여 이 등록을 수락하거나 거부하세요.
   reminder_body: >
-    This email is to serve as a friendly reminder that you have an open
-    task waiting for you at this time. Please login to %{site_link} at your
-    earliest convenience to complete what is available.
+    이 이메일은 현재 대기 중인 할 일이 있음을 상기시켜 드리기 위해 발송되었습니다.
```
Comment (mgmodell) [inline config/locales/home.ko.yml:113]: I cannot tell what this is attached to and it appears to be a duplication of the item below. Does this belong here?

**Review by mgmodell — CHANGES_REQUESTED** [review]
Assuming that the translated content itself is correct (and I cannot validate that), these additions do appear as though they would work if the comments I've provided are addressed.

**Pull request merged**